app/javascript/dashboard/routes/dashboard/settings/agents/EditAgent.vue

```diff
@@ -132,6 +143,22 @@ export default {
         this.showAlert();
       });
     },
+    resetPassword() {
+      // Call resetPassword from Auth Service
+      Auth.resetPassword(this.agentCredentials)
+        .then(res => {
+          this.editAgentsApi.message = this.$t('AGENT_MGMT.EDIT.PASSWORD_RESET.SUCCESS_MESSAGE');
+          this.showAlert();
+          this.resetForm();
+          setTimeout(() => {
```

Comment:
Why do we use setTimeouts?

Reply:
This is to close the EditAgent popup.

Reply:
@mukesh4139 I don't think we need to close the Popup on successful reset.